src/clr-angular/wizard/wizard.ts

```diff
@@ -393,7 +393,9 @@ export class ClrWizard implements OnDestroy, AfterContentInit, DoCheck {
    * alternative cancel functionality. In most cases, use `ClrWizard.cancel()` instead.
    */
   public modalCancel(): void {
-    this.checkAndCancel();
+    if (this.closable) {
+      this.checkAndCancel();
```

Comment:
Why wouldn't this check go into the `checkAndCancel` method?

Reply:
I wasn't sure about putting it into the `checkAndCancel` method because it is public and not used in the template. It seemed like a convenience method for developers to call. I was thinking if that was the case then calling `wizard.checkAndCancel()` programmatically should cancel the wizard regardless if the wizard is in a closable state.